vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/impl/PackagingImpl.java

```diff
@@ -106,6 +106,9 @@ public PackagingImpl() {
 
         @AttributeDefinition(description = "Whether to overwrite the primary type of folders")
         boolean overwritePrimaryTypesOfFolders() default true;
+
+        @AttributeDefinition(description = "Default IdConflictPolicy")
```

Comment:
Please use the enum values instead (not free-text String), that also prevents type conversion.

Reply:
For some reason I though this would be hard, but apparently it's not :-)